Resources/Prototypes/Entities/Structures/Doors/Windoors/assembly.yml

```
@@ -15,7 +15,7 @@
     fixtures:
     - shape:
         !type:PhysShapeAabb
-        bounds: "-0.2,-0.49,-0.49,0.49"
+        bounds: "-0.49,-0.49,0.49,-0.2"
```

wait what, shouldnt it be:
```suggestion
        bounds: "-0.49,-0.2,0.49,-0.49"
```